app/src/main/java/com/teamwss/websoso/data/di/DataStoreModule.kt

```diff
@@ -0,0 +1,39 @@
+package com.teamwss.websoso.data.di
+
+import android.content.Context
+import androidx.datastore.core.DataStore
+import androidx.datastore.core.handlers.ReplaceFileCorruptionHandler
+import androidx.datastore.preferences.SharedPreferencesMigration
+import androidx.datastore.preferences.core.PreferenceDataStoreFactory
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.core.emptyPreferences
+import androidx.datastore.preferences.preferencesDataStoreFile
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+import javax.inject.Singleton
+
+@InstallIn(SingletonComponent::class)
+@Module
```

Comment:
r: 다른 코드와 통일성 및 가독성을 위해 순서를 바꿔주시면 좋을 것 같아요

```suggestion
@Module
@InstallIn(SingletonComponent::class)
```

Reply:
아 못봤네요 감사합니당